_posts/2019-09-06-usda-linked-nutrition-data.md

```diff
@@ -0,0 +1,63 @@
+
+{:.post-image-box .post-image-box-center}
+![ ]({{ site.baseurl }}/assets/img/posts/2019-09-06-image001.png "Diagram of the files.")  
+
+## Probabilistic and Semantic Matching 
+
+The team created the linking database using a combination of auto¬mated and manual matches, with intermediate review by nutritionists. The final result was 650,592 UPCs matched to 4,390 Food and Nutrient Database for Dietary Studies ([Food and Nutrient Database for Dietary Studies (FNDDS)](https://data.nal.usda.gov/dataset/food-and-nutrient-database-dietary-studies-fndds)) and National Nutrient Database for Standard Reference (SR) with a 5-percent error rate for each linking category.
+
+The team  used semantic matching  to identify possible sub-text string matches between the federal and commercial data. Semantic matching searches full-text strings in one list for words and phrases in the other list that are either identical or mean similar things.
+
+Both automated semantic matching methods and human review developed the search table that paired IRI food description terms with USDA food descrip¬tion terms having the same meaning. Automated methods developed draft mapping rules, and then nutritionists reviewed all rules and augmented the search table by identifying phrases in the IRI text descriptions that match to FNDDS descriptions. 
+
+In probabilistic matching, a program used the search table to compare the attributes in each UPC text description and other information in the IRI data to FNDDS text descriptors. The similarity of the two food descriptions across a number of different attributes determined a similarity score for each possible match. Matches between attribute values (or synonyms) from the search table added to the total similarity score, while nonmatches subtracted from the score. The program selected IRI-FNDDS food item pairs with the highest score. 
+
+To use the power of the semantic and probabilistic matching, the data had to be prepared. Researchers prioritized which UPCs and USDA food codes were included, created complete text descriptions, and divided the UPCs and USDA food codes into linking categories to streamline the matching process.  For some linking categories, the team parsed the USDA text descriptions into columns  more similar to the IRI data. In other cases, it was more efficient to combine IRI fields into a single text string.
+
+## New Insights: Americans’ Store Food Purchases Are Not That Healthy
+
+{:.post-image-box .post-image-box-center}
+![ ]({{ site.baseurl }}/assets/img/posts/2019-09-06-image002.png "Retail food sales in the United States do not align with Federal dietary recommendations.")
```

Comment:
Can you add an alt tag here to help with accessibility tools, like screen readers?

```suggestion
![Bar chart showing the Healthy Eating Index (HEI-2015)]({{ site.baseurl }}/assets/img/posts/2019-09-06-image002.png "Retail food sales in the United States do not align with Federal dietary recommendations.")
```